python-threatexchange/threatexchange/extensions/video_vpdq/tests/test_vpdq_hash_and_match.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import pytest
+from pathlib import Path
+import sys
+
+sys.path.append(str(Path(__file__).parents[1]))
+try:
+    import vpdq as _  # type: ignore
+
+    _DISABLED = False
+except (ImportError, ModuleNotFoundError) as e:
+    _DISABLED = True
+else:
+    from threatexchange.extensions.video_vpdq.video_vpdq import VideoVPDQSignal
+    from vpdq_util import vpdq_to_json, read_file_to_hash, TARGET_MATCH_PERCENT, QUERY_MATCH_PERCENT  # type: ignore # Import from parent
```

Comment:
Why does this need type ignore? Why isn't this included in the extension itself?